ksql-cli/src/main/java/io/confluent/ksql/cli/console/JLineReader.java

```diff
@@ -97,7 +100,7 @@ public String expandVar(final String word) {
         .option(LineReader.Option.HISTORY_IGNORE_SPACE, false)
         .option(LineReader.Option.DISABLE_EVENT_EXPANSION, false)
         .expander(expander)
-        .parser(new TrimmingParser(parser))
+        .parser(new TrimmingParser(new TerminationParser(parser)))
```

Comment:
i _think_, if i remember correctly from my own adventures in jline-wonderland, that this pattern is going to cause all of the default parsing work - which is a considerable amount of string manipulation - to happen twice ? Maybe collapse the 2 custom parsers into one

Reply:
Hi @blueedgenick , I don't believe that's the case. The default parser is only called once.

Reply:
oh i see - the new one is the delegate of the previously added one, got it. Sorry for the noise!

Reply:
No worries Nick, the more eyes the better :D